explanation/service-endpoint-encryption.rst

```diff
@@ -41,3 +40,31 @@ Points of interest for this design:
    This updates endpoints with HTTPS and also distributes the CA
    certificates to all the application units across the cloud via
    Keystone.
+
+TLS Vault feature
+-----------------
+
+The TLS Vault feature is the method to use for deployments that use Vault as an intermediary CA.
+
+.. tip::
+   For a how-to on using the TLS Vault feature see :doc:`TLS Vault
+   </how-to/features/tls-vault>`.
```

Comment:
I would recommend removing the "how-to/features/" section altogether because I think it's confusing. I think a more straightforward how-to named "Manage secrets with Vault" would be preferable. It can even live in a second-level header called "Manage your deployment" or something similar.

Reply:
@hemanthnakkina I agree naming this better, but there is also a TLS CA file. Here we either rename both, or keep TLS Vault.
I don't agree to remove this section because the new user doesn't know where to search for this feature... It is much easier to have a link pointing out to it